.github/workflows/unit-tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
+        node-version: [18.x, 20.x]
```

Comment:
we still have 16 available in vercel, do we want to drop these tests? are they failing?

Reply:
Yep Next.js dropped Node 16 support in 14.x